worlds/Files.py

```diff
@@ -135,31 +126,61 @@ def read(self, file: Optional[Union[str, BinaryIO]] = None) -> None:
                         message = f"{arg0} - "
                 raise InvalidDataError(f"{message}This might be the incorrect world version for this file") from e
 
-    def read_contents(self, opened_zipfile: zipfile.ZipFile) -> None:
+    def read_contents(self, opened_zipfile: zipfile.ZipFile) -> Dict[str, Any]:
```

Comment:
Nitpicking: Now that the minimal python version is 3.10, using `Dict` from typing is no longer required. The return type could be replaced with `dict[str, Any]`. 

Same for the return type of `get_manifest` and the various `Optional[str]` that could be replaced with `str | None`.